tools/lib/provision.py

```diff
@@ -145,7 +149,7 @@
 ] + YUM_THUMBOR_VENV_DEPENDENCIES
 
 BUILD_PGROONGA_FROM_SOURCE = False
-if vendor == 'debian' and os_version in []:
+if "debian" in os_families() and os_version in ["20.04"]:
```

Comment:
I'd prefer to write this as ` vendor == 'debian' and os_version in [] or vendor == 'ubuntu' and os_version in ['20.04']:` so that we're thoughtful about namespacing `os_version` within vendors.  It'll do the same thing as long as Debian and Ubuntu never have overlapping version numbers, but good code avoids the developer needing to think about that sort of unnecessary mutable "unless" clause about the world.